tools/eubfr-cli/README.md

```diff
@@ -0,0 +1,30 @@
+# EUBFR CLI
+
+## Usage
+
+```sh
+$ npx eubfr-cli
+```
+
+If you want to make use of the CLI to automatically upload or delete all content of a given stage, you can optionally create a `.content` folder in the root of your project, with the following example structure:
+
+```
+.
+├── agri
+│   └── agri_history.csv
+├── budg
+│   └── CreativeEurope_Projects_Overview_2017-08-21.xls
+├── iati
+│   └── activity.csv
+├── inforegio
+│   ├── EUBFR_VIEW_16052018.xml
+│   └── regio_projects.json
+├── valor
+│   └── valor_sample.xls
+└── wifi4eu
+    └── wifi4euRegistrations.xlsx
+```
+
+Then either run `yarn content:upload` to upload of that content to their respective S3 buckets which trigger the ingestion process for each producer, or run `yarn content:delete` to delete all the currently uploaded content of a given stage.
+
```

Comment:
Shouldn't we add the few example you wrote in PR's description here?
It helps to understand how to use the CLI.

Reply:
Sure, I'll include further details.